exchanges/okx/okx_wrapper.go

```diff
@@ -131,27 +131,27 @@ func (ok *Okx) SetDefaults() {
 			AutoPairUpdates: true,
```

Comment:
The automated testing process for retrieving historical candle data, known as TestGetHistoricCandles, is currently experiencing a state in which it is unable to meet the desired outcome. This failure could be the result of a variety of factors, such as an error in the code, an issue with the data source, or an external factor that is disrupting the testing process. Regardless of the cause, this failure signifies that the expected results are not being achieved, and the desired functionality of retrieving historical candle data is not being fulfilled at this time. It is important to identify and address the underlying cause of this failure promptly to ensure the proper functioning of this critical process.

 okx_test.go:2309: candle count exceeded: 226. The endpoint has a set candle limit return of 100 candles. Candle data will be incomplete: request will exceed exchange limits, please reduce start-end time window or use GetHistoricCandlesExtended